projects/components/src/dropdown/dropdown-focus-handler.directive.ts

```diff
@@ -159,10 +169,18 @@ export class DropdownFocusHandlerDirective<T> implements AfterViewInit, OnDestro
 
     private linkMenuItems(): void {
         const menuChildren: Element[] = Array.from(this.clrDropdownMenuEl.children);
-        this.menuItems = menuChildren.map((child) => ({
-            element: (child.matches('button') ? child : child.querySelector('clr-dropdown > button')) as HTMLElement,
-            left: this.menuTrigger,
-        }));
+        this.menuItems = menuChildren
+            .filter(
+                (child) =>
+                    child.nodeName === DropdownMenuItemType.BUTTON ||
+                    child.nodeName === DropdownMenuItemType.NESTED_DROPDOWN_TRIGGER
+            )
+            .map((child) => ({
+                element: (child.matches('button')
```

Comment:
Should we make the part after `element:` a separate function? Call it something like `getDropdownButton(child)`?